### PR TITLE
Run tests in no debug mode

### PR DESCRIPTION
**Pull request by yguedidi**
From [4m04s](https://github.com/wallabag/wallabag/actions/runs/8018668664) on latest master CI run to now [2m51s](https://github.com/wallabag/wallabag/actions/runs/8027285194): ~ -1m10s

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -125,6 +125,9 @@ fos_rest:
             # for an unknown reason, EACH REQUEST goes to FOS\RestBundle\EventListener\FormatListener
             # so we need to add custom rule for custom api export but also for all other routes of the application...
             - { path: '^/', priorities: ['text/html', '*/*'], fallback_format: html, prefer_extension: false }
+    exception:
+        messages:
+            Symfony\Component\HttpKernel\Exception\BadRequestHttpException: true
```
Comment (yguedidi) [inline app/config/config.yml:130]: add to preserve the "API limit reached" error message in final JSON representing the exception

File: src/Controller/Api/EntryRestController.php [hunk 2]
```diff
@@ -528,7 +528,7 @@ public function postEntriesListAction(Request $request, EntryRepository $entryRe
         $limit = $this->getParameter('wallabag.api_limit_mass_actions');
 
         if (\count($urls) > $limit) {
-            throw new HttpException(400, 'API limit reached');
+            throw new BadRequestHttpException('API limit reached');
```
Comment (yguedidi) [inline src/Controller/Api/EntryRestController.php:531]: changed to be more specific as needed to be allowed in config to preserve the message

File: tests/Controller/ConfigControllerTest.php [hunk 3]
```diff
@@ -1313,7 +1313,8 @@ public function testImportTagginfRuleFile()
         $client->submit($form, $data);
         $this->assertSame(302, $client->getResponse()->getStatusCode());
 
-        $user = $client->getContainer()->get('fos_user.user_manager.test')->findUserBy(['username' => 'admin']);
+        $user = static::getContainer()->get('fos_user.user_manager')->findUserBy(['username' => 'admin']);
+        \assert($user instanceof User);
```
Comment (yguedidi) [inline tests/Controller/ConfigControllerTest.php:1317]: needed for PHPStan as the UserManager returns an instance of UserInterface

File: phpstan.neon [hunk 4]
```diff
@@ -8,7 +8,7 @@ parameters:
         - tests
 
     symfony:
-        container_xml_path: %rootDir%/../../../var/cache/test/AppKernelTestDebugContainer.xml
+        container_xml_path: %rootDir%/../../../var/cache/dev/AppKernelDevDebugContainer.xml
```
Comment (yguedidi) [inline phpstan.neon:11]: XML file is not generated in no debug so switch to dev env

**Comment by j0k3r** [comment]
Conflicts

**Pull request merged**